tests/test_accuracy_comprehensive.py

```
@@ -0,0 +1,253 @@
+                    failed_cases.append(f"Electricity: {text[:40]}... got {result.get('electricity_kwh')}, expected {expected_elec}")
+            
+            if expected_carbon is not None:
+                total_fields += 1
+                if result.get("carbon_kgco2e") == expected_carbon:
+                    correct_fields += 1
+                else:
+                    failed_cases.append(f"Carbon: {text[:40]}... got {result.get('carbon_kgco2e')}, expected {expected_carbon}")
+        
+        accuracy = correct_fields / total_fields * 100
+        
+        print(f"\n=== GROUND TRUTH ACCURACY REPORT ===")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)